src/backup.ts

```diff
@@ -243,6 +253,45 @@ Please use the format 'my-backup' or '${cluster.name}/backups/my-backup'.`);
     });
   }
 
+  /**
+   * Copy this backup
+   *
+   * @param {CopyBackupConfig} [config] The config that specifies all of the
+   * information about the destination backup.
```

Comment:
what is the "destination backup"? Is that the backup that will be copied, or the backup to be copied? Please add some more info here.

Reply:
I added some more information here about this parameter.